### ISSUE TITLE
Cannot Override CSS of the 'Callout' Fabric Component

### ISSUE THREAD

**Issue by tracy-zaact** [SharePoint/sp-dev-docs#1490]
#### Category
- [ ] Question
- [ ] Typo
- [x] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

For other Fabric components (such as the IconButton), I can override the CSS by applying a className with a custom class from my scss file like indicated below.  In this case, I can see the styles are properly applied when viewing the dev console.  

`                                <IconButton
                                disabled={ false }
                                onClick={ this.toggleScoreCallOut }
                                iconProps={ { iconName: 'Help' } }
                                title='Overall Project Score Explanation'
                                ariaLabel='Overall Project Score Explanation'
                                className="execCallout"
                                />   `

#### Observed Behavior

However, when I try the exact same operation on a Callout component within the same SPFx web part - it does not work.  I am able to see that 'execCallout' is added to the class attribute of the relevant mark-up, but the class is not associated with the styles I provided in the .scss file.  They do not appear when looking in the Styles tab of the dev console.  

`                                <Callout
                                role={ 'alertdialog' }
                                gapSpace={ 0 }
                                target={ this.scoreCalloutElement }
                                onDismiss={ this.scoreCallOutDismiss }
                                setInitialFocus={ true }
                                className="execCallout"
                                >`

#### Steps to Reproduce

1. Add a [Callout Fabric component](https://developer.microsoft.com/en-us/fabric#/components/callout) to an SPFx web part.  
2. Provide a custom class to the element's className attribute with the following:  `max-width: 300px;  `.
1. Observe whether the style is applied, and appears within the Styles referenced in the dev console.

**Comment by tracy-zaact** [comment]
Possibly related to this issue:  https://github.com/OfficeDev/office-ui-fabric-react/issues/445?

**Comment by tracy-zaact** [comment]
Figured it out with the help of a colleague.  Because the Callout HTML element is dynamically created outside the web part's parent scoping element, any classes defined within the top-level wrapper do not get applied.  Once I moved my class outside that top-level wrapper, it is being applied now.  Unfortunately, this means we cannot take advantage of the CSS scoping functionality on dynamic elements such as these.
